docs/design/time.md

```diff
@@ -0,0 +1,77 @@
+---
+sidebar_position: 6
+---
+
+# Time
+
+Time is a critical component of all distributed systems and databases. Within TigerBeetle, we keep
+track of two types of time: logical time and physical time.
+
+## Logical Time
+
+TigerBeetle uses [Viewstamped replication](https://pmg.csail.mit.edu/papers/vr-revisited.pdf) for
+its consensus protocol. This ensures [strict
+serializability](http://www.bailis.org/blog/linearizability-versus-serializability/) for all
+operations.
+
+## Physical Time
+
+TigerBeetle uses physical time in addition to the logical time provided by the consensus algorithm.
+Financial transactions require physical time for multiple reasons, including:
+
+- **Liquidity** - TigerBeetle supports [Two-Phase Transfers](./two-phase-transfers.md) that reserve funds
+  and hold them in a pending state until they are posted, voided, or the transfer times out. A
+  timeout is useful to ensure that the reserved funds are not held in limbo indefinitely.
+- **Compliance and Auditing** - For regulatory and security purposes, it is useful to have a
+  specific idea of when (in terms of wall clock time) transfers took place.
+
+### Physical Time is Propagated Through Consensus
+
+TigerBeetle clusters are distributed systems, so we cannot rely on all replicas in a cluster keeping
+their system clocks perfectly in sync. Because of this, we "trust but verify".
+
+In the TigerBeetle consensus protocol messages, replicas exchange timestamp information and monitor
+the skew of their clock with respect to the other replicas. The system ensures that the clock skew
+is within an acceptable window and may choose to stop processing transactions if the clocks are far
+out of sync.
+
+Importantly, the goal is not to reimplement clock synchronization protocols, but to verify that the
+cluster is operating within acceptable error bounds.
```

Comment:
```suggestion
Importantly, the goal is not to reimplement or replace clock synchronization protocols, but to verify that the
cluster is operating within acceptable error bounds.
```

to clarify that operators should still run NTP on replica servers, not just rely on TB to figure it out. (We could spell that out explicitly too, but that belongs more in a deployment guide than here.)